repos/system_upgrade/common/models/ipuconfig.py

```diff
@@ -33,6 +33,21 @@ class Version(Model):
     target = fields.String()
     """Version of the target system. E.g. '8.2.'."""
 
+    virtual_source_version = fields.String()
+    """
+    Source OS version used when checking whether to execute version-dependent code.
+
+    On RHEL and other systems that have version of the form MINOR.MAJOR, `virtual_source_version`
+    matches `source_version`.
+
+    CentOS has version of the form MAJOR, lacking the minor version number. The
+    `virtual_source_version` value is obtained by combining CentOS major
+    version number with a minor version number stored internally in leapp.
```

Comment:
This could be confusing as there is not such a value stored in leapp (project).

```suggestion
    version number with a minor version number stored internally in the upgrade_paths.json file.
```